usage.c

```diff
@@ -15,7 +15,12 @@ void vreportf(const char *prefix, const char *err, va_list params)
 
 	fflush(fh);
 	if (!tweaked_error_buffering) {
+#if defined(_MSC_VER)
+		/* UCRT doesn't like zero buffer size */
+		setvbuf(fh, NULL, _IOLBF, BUFSIZ);
+#else
 		setvbuf(fh, NULL, _IOLBF, 0);
+#endif
```

Comment:
You mean:
> If buffer set to a null pointer, the function automatically allocates a buffer.

But the documentation does not say that the function supports the zero size, so _IOLBF should be always with non-zero size as you suggesting.